.changelog/DEV-1787.yaml

```diff
@@ -0,0 +1,16 @@
+# Type of change
+change_type: enhancement
+
+# The name of the component
+component: feature
```

Comment:
Should we change this to target?

Reply:
oh yea, i confused this as "new feature" 😅